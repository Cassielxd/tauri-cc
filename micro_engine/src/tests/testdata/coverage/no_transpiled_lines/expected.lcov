SF:[WILDCARD]index.ts
FNF:0
FNH:0
BRF:0
BRH:0
DA:1,1
DA:2,1
DA:3,1
DA:5,1
LH:4
LF:4
end_of_record
