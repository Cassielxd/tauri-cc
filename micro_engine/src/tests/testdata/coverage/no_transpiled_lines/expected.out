cover [WILDCARD]index.ts ... 100.000% (4/4)
